nvflare/private/fed/runner.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Runner:
+    def start(self):
+        """Method call at the start of the Runner process."""
+        pass
+
+    def stop(self):
+        """Method call at the end of the Runner process."""
```

Comment:
If only `stop` method is needed, suggest

```
class Runner(ABC):
    @abstractmethod
    def stop(self):
        """Stops the runner."""
        pass
```

Reply:
changed.